Packs/Jira/Integrations/JiraV3/JiraV3_test.py

```diff
@@ -2668,3 +2668,87 @@ def test_test_update_issue_assignee_command_no_assignees(self):
 
         with pytest.raises(DemistoException):
             update_issue_assignee_command(client=client, args=args)
+
+
+class TestJiraIssueGetForms:
+    @pytest.mark.parametrize(
+        'issue_id',
+        [
+            ("TES-2")
```

Comment:
```suggestion
            ("TEST-2")
```
Did you mean `TEST-2`?